### DISCUSSION TITLE
Vehicle stock

### DISCUSSION THREAD

**Discussion by paula-fel** [transport-data/community#6] [Data needs]
For Vehicle Stock we desire data with these dimensions

- vehicle type
- fuel type
- age

Some sources that, per scoping, provide data for this measure: 

- #10 
- #11 
- #12 
- #13 
- #14 
- #7 
- #8

**Comment by khaeru** [comment]
During a TDCI meeting today, several people pointed out that official sources (e.g. national governments) often have an issue with **not removing de-registered cars** (i.e. passenger road vehicles in particular) that are scrapped, exported, no longer in use, etc. from counts of stocks.

Have you run into such issues? How did you detect, address, and correct it?

**Comment by khaeru** [comment]
Just a note that **vehicle sales** can be understood as _additions to stock_ in a given period; the thing measured and the units of measurement are the same.

Using SDMX, there could be multiple ways to represent this:
- Separate concepts/measure for `STOCK` and `SALES` (and retirements, and exports, etc.)
- A single concept/measure for `STOCK`, together with an concept/attribute or dimension indicating whether the data are:
  - Total stock of vehicles in-use as of a point in time,
  - Additions to stock in a period of time (i.e. “sales”),
  - etc.

**Reply by khaeru** [reply]
#21 from @icct-emulholland reminds that **registrations** is another term related to additions to stock. Precisely, I suppose a vehicle can be sold in a particular region yet not registered, e.g. immediately exported to a different region.
